scripts/bulk_index.pl

```diff
@@ -49,32 +49,40 @@ BEGIN
 
 $indexer->create_index;
 
+my $run_time = time;
 my $loop = IO::Async::Loop->new;
 my $timer = IO::Async::Timer::Periodic->new(
     first_interval => 0,
     interval       => 15,
     reschedule     => 'skip',
 
     on_tick => sub {
+        printf "Running after %d seconds\n", time - $run_time;
         my $start_users = time;
         say "indexing users" if $verbose;
```

Comment:
Per the immediately-following review comment, rather than printing "indexing users" here, you could just do it as part of the bulk_load('Bugzilla::User') call. You already have the string 'Bugzilla::User' handy to emit in a log message, and that routine should be responsible for logging its own stats anyways IMO.